src/components/players/details/recentGamesStats/index.js

```diff
@@ -61,14 +61,14 @@ const RecentGamesStats = ({ teams, recentGamesStats }) => {
             <div className={styles.date}>{formatUTCDate(gameDateUTC)}</div>
```

Comment:
While we're here, I think design wise I want the date to have a little more color. The reasoning being that the 3 games listed don't really have something to make them separate them (design wise). It just looks like a bunch of text. So if we change the date to have the `teamColor`, I think it'll look much better:

```
<div style={{ color : teamColor }} className={styles.date}>{formatUTCDate(gameDateUTC)}</div>
```

---

**raptors**:

<img width="449" alt="Screen Shot 2020-05-15 at 3 50 15 PM" src="https://user-images.githubusercontent.com/623755/82090550-cb310d00-96c3-11ea-9853-000d9cd5bc92.png">

---

**magic**:

<img width="450" alt="Screen Shot 2020-05-15 at 3 51 28 PM" src="https://user-images.githubusercontent.com/623755/82090656-f87dbb00-96c3-11ea-8e1c-98722f5be814.png">

---

**jazz**:

<img width="448" alt="Screen Shot 2020-05-15 at 3 53 55 PM" src="https://user-images.githubusercontent.com/623755/82090838-4e526300-96c4-11ea-94e8-122e853cfeed.png">

Reply:
thoughts?